scripts/jenkins/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+# Install Go using the same travis approach
+echo "Installing ${GO_VERSION} with gimme."
+eval "$(curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | GIMME_GO_VERSION=${GO_VERSION} bash)"
```

Comment:
test.sh is only run after before_install/build.sh, so this is redundant isn't it?